digdag-server/src/main/java/io/digdag/server/rs/ProjectResource.java

```diff
@@ -204,6 +206,8 @@ public ProjectResource(
         this.scsp = scsp;
         this.projectArchiveLoader = projectArchiveLoader;
         MAX_SESSIONS_PAGE_SIZE = systemConfig.get("api.max_sessions_page_size", Integer.class, DEFAULT_SESSIONS_PAGE_SIZE);
+        MAX_ARCHIVE_TOTAL_SIZE_LIMIT = systemConfig.get("api.max_archive_total_size_limit", Integer.class, DEFAULT_ARCHIVE_TOTAL_SIZE_LIMIT);
+        MAX_ARCHIVE_FILE_SIZE_LIMIT = MAX_ARCHIVE_TOTAL_SIZE_LIMIT;
```

Comment:
Should use `MAX_ARCHIVE_FILE_SIZE_LIMIT` as default value.

Reply:
yes, that makes more sense.